apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/repository/AccessKeyRepositoryTest.java

```diff
@@ -0,0 +1,66 @@
+package com.ctrip.framework.apollo.biz.repository;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.ctrip.framework.apollo.biz.AbstractIntegrationTest;
+import com.ctrip.framework.apollo.biz.entity.AccessKey;
+import java.util.Date;
+import java.util.List;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
+
+
+public class AccessKeyRepositoryTest extends AbstractIntegrationTest {
+
+  @Autowired
+  private AccessKeyRepository accessKeyRepository;
+
+  @Test
+  public void testSave() {
+    String appId = "someAppId";
+    String secret = "someSecret";
+    AccessKey entity = new AccessKey();
+    entity.setAppId(appId);
+    entity.setSecret(secret);
+
+    AccessKey accessKey = accessKeyRepository.save(entity);
+
+    assertThat(accessKey).isNotNull();
+    assertThat(accessKey.getAppId()).isEqualTo(appId);
+    assertThat(accessKey.getSecret()).isEqualTo(secret);
+  }
+
+  @Test
+  @Sql(scripts = "/sql/accesskey-test.sql", executionPhase = ExecutionPhase.BEFORE_TEST_METHOD)
+  @Sql(scripts = "/sql/clean.sql", executionPhase = ExecutionPhase.AFTER_TEST_METHOD)
+  public void testFindByAppId() {
+    String appId = "someAppId";
+
+    List<AccessKey> accessKeyList = accessKeyRepository.findByAppId(appId);
+
+    assertThat(accessKeyList).hasSize(1);
+    assertThat(accessKeyList.get(0).getAppId()).isEqualTo(appId);
+    assertThat(accessKeyList.get(0).getSecret()).isEqualTo("someSecret");
+  }
+
+  @Test
+  @Sql(scripts = "/sql/accesskey-test.sql", executionPhase = ExecutionPhase.BEFORE_TEST_METHOD)
+  @Sql(scripts = "/sql/clean.sql", executionPhase = ExecutionPhase.AFTER_TEST_METHOD)
+  public void testFindFirst500ByDataChangeLastModifiedTimeGreaterThanOrderByDataChangeLastModifiedTime() {
+    // 2019-12-19 10:46:35
+    Date date = new Date(1576723194000L);
```

Comment:
This causes the ci failed, because `1576723194000L` is 2019-12-19 10:39:54 in UTC+8, which is 2019-12-19 02:39:54 in UTC+0, so there will be 3 records returned instead of 2.

You may consider to use Calendar to create the date instance.

Reply:
Thank you, and I think I can use `Instant` to create `Date`, like this:
```
Instant instant = LocalDateTime.of(2019, 12, 19, 10, 39, 54)
    .atZone(ZoneId.systemDefault())
    .toInstant();
Date date = Date.from(instant);
```